experimenter/execute_pipeline.py

```diff
@@ -1,218 +0,0 @@
-"""
```

Comment:
Are you sure we don't need this file anymore?

Reply:
I changed the name to execute_pipeline_old.py. For some reason, the name changes were not committed. I just fixed that. They should be there now. (two new files, runtime.py and execute_pipeline_old.py)